crates/hdi/CHANGELOG.md

```diff
@@ -7,6 +7,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## Unreleased
 
+- Change the license from Apache-2.0 to CAL-1.0 to match the HDK.
```

Comment:
i don't remember the reason we chose a different license for the HDI.

@artbrock @thedavidmeister either of you please chime in here.

Reply:
In order to build a useful zome, you need the HDK so any software you produce is going to be restricted by the CAL anyway. I guess it'd be possible to write a library based on the HDI that is used in integrity zomes only and maybe you'd want a different license for some reason. But the end purpose would still be use in Holochain with the HDK as far as I can imagine. I'm happy to revisit this if necessary but I don't believe it's restrictive to use the CAL license there.